src/bin/__test__/cli.test.ts

```diff
@@ -40,12 +42,80 @@ test('cli: duplicate option', async () => {
   expect(logger.error).toHaveBeenCalledWith('Invalid options. Singular option already set [help=true]')
 })
 
+test('cli: unrecognised command', async () => {
+  await run(['xyz'])
+  expect(logger.error).toHaveBeenCalledWith('Unrecognized command "xyz".')
+})
+
+test('cli: unrecognised command', async () => {
+  await run(['--opt'])
+  expect(logger.error).toHaveBeenCalledWith('Command expected, nothing provided.')
+})
+
+// NODE
+
 test('cli: upload-node command', async () => {
   const logSpy = jest.spyOn(global.console, 'log').mockImplementation(() => {})
   run(['upload-node', '--help'])
-  expect(logSpy).toHaveBeenCalledWith(expect.stringContaining('TODO'))
+  expect(logSpy).toHaveBeenCalledWith(expect.stringContaining('bugsnag-source-maps upload-node'))
+})
+
+test('cli: upload-node invalid option', async () => {
+  await run(['upload-node', 'foo'])
+  expect(logger.error).toHaveBeenCalledWith('Invalid argument provided. Unknown value: foo')
 })
 
+test('cli: upload-node exit code for failure', async () => {
+  const mockedUpload = node.uploadOne as jest.MockedFunction<typeof node.uploadOne>
+  mockedUpload.mockRejectedValue(new Error('fail'))
+  await run(['upload-node', '--api-key', '123', '--source-map', 'bundle.js.map'])
+  expect(process.exitCode).toBe(1)
+})
+
+test('cli: upload-node single mode', async () => {
+  await run(['upload-node', '--api-key', '123', '--bundle', 'bundle.js', '--source-map', 'bundle.js.map'])
+  expect(node.uploadOne).toHaveBeenCalledTimes(1)
+  expect(node.uploadOne).toHaveBeenCalledWith(
+    expect.objectContaining({ apiKey: '123', bundle: 'bundle.js', sourceMap: 'bundle.js.map' })
+  )
+})
+
+test('cli: upload-node multiple mode', async () => {
+  await run(['upload-node', '--api-key', '123', '--directory', 'dist'])
+  expect(node.uploadMultiple).toHaveBeenCalledTimes(1)
+  expect(node.uploadMultiple).toHaveBeenCalledWith(
+    expect.objectContaining({ apiKey: '123', directory: 'dist' })
+  )
+})
+
+test('cli: upload-node single mode missing opts', async () => {
+  await run(['upload-node', '--api-key', '123', '--bundle', 'http://my.url/dist/bundle.js'])
+  expect(process.exitCode).toBe(1)
+  expect(logger.error).toHaveBeenCalledWith('--source-map is required')
+
+  await run(['upload-node', '--api-key', '123'])
+  expect(process.exitCode).toBe(1)
+  expect(logger.error).toHaveBeenCalledWith('Not enough options supplied')
+
+  await run(['upload-node'])
```

Comment:
Setting `process.exitCode` to 0 between these tests (or splitting them into separate tests) would be good, so we know the exit code is actually being changed